lib/workers/repository/init/index.ts

```diff
@@ -54,6 +54,11 @@ export async function initRepo(
   await initializeCaches(config as WorkerPlatformConfig);
   config = await getRepoConfig(config);
   setRepositoryLogLevelRemaps(config.logLevelRemap);
+  if (config.mode === 'silent') {
+    logger.info(
+      'Repository is running with mode=silent and will not make Issues or PRs by default',
+    );
```

Comment:
This INFO level messages when in silent mode should hopefully be more clear to users of the Mend app than currently where dryRun=lookup is debug and non-obvious